src/_pytest/compat.py

```diff
@@ -4,7 +4,7 @@
 import inspect
 import os
 import sys
-from contextlib import contextmanager
+from contextlib import nullcontext as nullcontext  # noqa: F401
```

Comment:
this should get inlined in the calling modules since there's no longer need for it in `compat`